<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Files</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="ListFileEdit" name="listFiles">
                                    <property name="acceptDrops">
                                        <bool>true</bool>
                                    </property>
                                    <property name="dragEnabled">
                                        <bool>true</bool>
                                    </property>
                                    <property name="dragDropMode">
                                        <enum>QAbstractItemView::DragDrop</enum>
                                    </property>
                                    <property name="defaultDropAction">
                                        <enum>Qt::MoveAction</enum>
                                    </property>
                                    <property name="alternatingRowColors">
                                        <bool>true</bool>
                                    </property>
                                    <property name="selectionMode">
                                        <enum>QAbstractItemView::ExtendedSelection</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QProgressBar" name="progressBar">
                                    <property name="value">
                                        <number>0</number>
                                    </property>
                                    <property name="invertedAppearance">
                                        <bool>false</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="Line" name="line">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Input files</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnBrowse">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Browse files</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_2">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>List actions</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QGridLayout" name="gridLayout">
                                    <item row="0" column="1">
                                        <widget class="QPushButton" name="btnDeleteAll">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Delete all</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="0">
                                        <widget class="QPushButton" name="btnDelete">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Delete</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="0">
                                        <widget class="QPushButton" name="btnDuplicate">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Duplicate</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="Line" name="line_3">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_5">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Output format</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="comboFormat">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <item>
                                        <property name="text">
                                            <string>JPEG</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>PNG</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_4">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnGetImages">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Get images</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <spacer name="horizontalSpacer_2">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeType">
                                                <enum>QSizePolicy::Preferred</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnBack">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Back</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>ListFileEdit</class>
            <extends>QListWidget</extends>
            <header>ListFileEdit</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
